contracts/governance/Vesting/TeamVesting.sol

```diff
@@ -1,29 +1,51 @@
 pragma solidity ^0.5.17;
 pragma experimental ABIEncoderV2;
 
-import "./Vesting.sol";
+import "../../openzeppelin/Ownable.sol";
```

Comment:
I'd recommend to manage OZ code as a dependency (via NPM). 
btw, the project doesn't include `package-lock.json`...
these notes are arbitrary placed in this PR (could be in any other)

Reply:
I've added OZ npm package initially,
but we have contracts in `openzeppelin` folder historically and
working together with these contracts and contracts from npm package is also not so good.
So we agreed to use contracts from `openzeppelin` folder.